frontend/src/main.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { createRoot } from "react-dom/client";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import routes from "@/misc/routing";
```

Comment:
Does `index.css` need to be imported here or in `index.html`?

Reply:
Usually an import in the html would be a better option, since it's better not to pollute this section in a tailwind project.

Reply:
Can we add it there then? Currently is not linked - https://github.com/club-40/odin-pantry/blob/7cfea69326adaf6ffbae8085f278fd49f82498f2/frontend/index.html

Reply:
Apparently tailwind needs it to be imported. Well that helps fix one problem.